schema-conformance/action.yaml

```diff
@@ -89,9 +89,11 @@ runs:
     - id: map-outputs
       if: always()
       run: |
-        echo "::set-output name=success::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests_succ }}"
-        echo "::set-output name=failed::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests_fail }}"
-        echo "::set-output name=error::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests_error }}"
-        echo "::set-output name=skip::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests_skip }}"
-        echo "::set-output name=total::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests }}"
+        echo "${{ steps.publish-junit.outputs.json }}"
+        cat $PWD/reports/junit.xml
```

Comment:
Are you planning to remove this later? Maybe we can move this under `debug: true` condition?